pyvrp/cpp/ProblemData_bindings.cpp

```diff
@@ -8,6 +8,13 @@ namespace py = pybind11;
 PYBIND11_MODULE(_ProblemData, m)
 {
     py::class_<ProblemData::Client>(m, "Client")
+        .def(py::init<int, int, int, int, int, int>(),
+             py::arg("x"),
+             py::arg("y"),
+             py::arg("demand") = 0,
+             py::arg("service_duration") = 0,
+             py::arg("tw_early") = 0,
+             py::arg("tw_late") = 0)
```

Comment:
Can we not default these? I don't know what appropriate defaults for e.g. the time windows should be, but [0, 0] does not seem to me to be 'it'.

@leonlan what do you think?

Reply:
I'm going to try `tw_late = 0` and an all-zero duration matrix in a minute. If that works without issues I think "everything zero" is a reasonable default.

Reply:
Seems to work well!